### PR TITLE
Create card

### PR DESCRIPTION
**Pull request by matthewdking**
relates #43 #6 #28 

- create a card component
- create button component

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: node_js
 node_js: 8.4.0
+notifications:
+  email: false
```
Comment (finnhodgkin) [inline .travis.yml]: Godsend :100:

File: src/containers/Careers.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
 import React, { Component } from 'react';
 import styled from 'styled-components';
+import Card from '../components/Card';
+import LikeAndDislikeButtons from '../components/LikeAndDislikeButtons';
 
 const CareersContainer = styled.div`
-  color: blue;
+  background-color: white;
+  padding-top: 1%;
+  height: 90vh;
```
Comment (finnhodgkin) [inline src/containers/Careers.js]: I'll have a think about other ways to do this than 90/10vh. I think giving the navbar an arbitrary percent height just to keep the content filling the remainder is not a perfect solution.

File: src/components/NavBar.js [hunk 3]
```diff
@@ -23,6 +24,7 @@ const StyledLink = styled(Link)`
 `;
 
 const Icon = styled.i`
+  color: white;
```
Comment (finnhodgkin) [inline src/components/NavBar.js]: lol big changes.

File: .travis.yml (hunk 1 above)
Comment (matthewdking) [reply .travis.yml]: :heart:

File: src/components/Card.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import styled from 'styled-components';
+
+const CardContainer = styled.div`
+  background-color: white;
+  height: 70%;
+  margin: 2rem 2rem 0;
+  overflow: auto;
+  border: 0.15rem solid gainsboro;
+  border-width: thin;
+  border-radius: 3%;
```
[the remark this answers is not in the document]
Comment (matthewdking) [reply src/components/Card.js]: Sure thing

File: src/components/LikeAndDislikeButtons.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  margin: 9% 5% 7%;
```
[the remark this answers is not in the document]
Comment (matthewdking) [reply src/components/LikeAndDislikeButtons.js]: I love my buggy solution hehe but sure will change it haha

File: src/components/NavBar.js [hunk 6]
```diff
@@ -4,17 +4,18 @@ import { Link } from 'react-router-dom';
 
 const NavBarContainer = styled.div`
   width: 100%;
-  height: 3.25rem;
+  height: 10vh;
```
[the remark this answers is not in the document]
Comment (matthewdking) [reply src/components/NavBar.js]: did calc instead not sure about having flex for every component

**Comment by matthewdking** [comment]
@finnhodgkin I think I have done everything! The font choice was restricted because it has to be available for Hebrew. We will have to look into Arabic fonts another time as well

**Comment by finnhodgkin** [comment]
LGTM :+1:

**Pull request merged**